CHANGELOG/CHANGELOG-3.6.md

```diff
@@ -8,7 +8,7 @@ Previous change logs can be found at [CHANGELOG-3.5](https://github.com/etcd-io/
 
 <hr>
 
-## v3.6.0-rc.0 (2025-02-13)
+## v3.6.0-rc.1 (TBD)
```

Comment:
Hi @ahrtr @ivanvc 

please let me know if it's not correct. Thanks

Reply:
updated.

Just curiosity: after offical release, we will merge all the rc changelogs into one, right?

Reply:
> after offical release, we will merge all the rc changelogs into one, right?

Probably not. If there isn't any new change in the following 2-3 weeks, then we may release v3.6.0.

Reply:
Thanks for the update.